src/breaking-change.ts

```diff
@@ -90,34 +80,42 @@ async function runOad(oldSpec: string, newSpec: string) {
   return JSON.parse(result);
 }
 
-/**
- * Processes the given swagger and stores the resolved swagger on to disk
- *
- * @param swaggerPath Path to the swagger specification file.
- */
-async function processViaAutoRest(swaggerPath: string) {
-  if (swaggerPath === null || swaggerPath === undefined || typeof swaggerPath.valueOf() !== 'string' || !swaggerPath.trim().length) {
-    throw new Error('swaggerPath is a required parameter of type "string" and it cannot be an empty string.');
-  }
+//main function
+export async function runScript() {
+  // Used to enable running script outside TravisCI for debugging
+  const isRunningInTravisCI = process.env.TRAVIS === 'true';
+  const outputFolder = path.join(os.tmpdir(), "resolved");
```

Comment:
why "resolved"? should it be something like "breakingChangesOutput"?

Reply:
Honestly, no idea. We may change it after merge.